Gaming The Syllabus/Assets/Scripts/XmlUtilities.cs

```diff
@@ -16,15 +16,30 @@ public static void Serialize(object item, string filePath)
 
     public static T Deserialize<T>(string filePath)
     {
+        Debug.Log(filePath);
+
         XmlSerializer serializer = new XmlSerializer(typeof(T));
         // Worth remembering that Path.Combine will just return the 2nd filePath if it is absolute (starts with a /)
-        StreamReader reader = new StreamReader(Path.Combine(Application.dataPath, filePath));
+        StreamReader reader = new StreamReader(Path.Combine(Application.persistentDataPath, filePath));
```

Comment:
Good call on using persistentDataPath to ensure important data such as user progress is saved in between play sessions.